validator/vm_external_ip.rego

```diff
@@ -36,10 +36,8 @@ deny[{
 	count(access_config) > 0
 
 	# Check if instance is in blacklist/whitelist
-	target_instances := params.instances
-	matches := {asset.name} & cast_set(target_instances)
-	target_instance_match_count(params.mode, desired_count)
-	count(matches) == desired_count
+trace(	sprintf("asset: %v, instances: %v, mode: %v", [asset.name, params.instances, params.mode]))
+	any_name_match(asset.name, params.instances, params.mode)
```

Comment:
Could we call this something like instance_name_targeted?

Reply:
changed the function name